src/SelfServe/Decorators.tsx

```diff
@@ -1,5 +1,9 @@
-import { ChoiceItem, Description, Info, InputType, NumberUiType, SmartUiInput, RefreshParams } from "./SelfServeTypes";
-import { addPropertyToMap, DecoratorProperties, buildSmartUiDescriptor } from "./SelfServeUtils";
+/**
+ * @module SelfServe/Decorators
```

Comment:
>Decorators [](http://example.com/codeflow?start=21&length=10)

Add a page/section called what is supported?

We have integrated support for 
1) portal notifications.
2) Telemetry
3) the following UI controls:
define each ui element supported with image of it..
info bubble, spinner,,etc..